lib/advSearchOk.ml

```diff
@@ -121,6 +138,8 @@ end = struct
 end
 
 module AdvancedSearchMatch : sig
+  type place = string * (string * Gwdb.istr option)
```

Comment:
If I am not mistaken it could just be a pair (the abbreviated name and the ID of the place)?